apps/src/componentLibrary/segmentedButtons/SegmentedButtons.story.tsx

```diff
@@ -0,0 +1,310 @@
+    },
+  ],
+};
+
+export const SizesOfSegmentedButtons = MultipleTemplate.bind({});
+SizesOfSegmentedButtons.args = {
+  components: [
+    {
+      buttons: [
+        {label: 'Label', value: 'label-xs'},
+        {
+          label: 'Another label',
+          value: 'another-label-xs',
+        },
+        {
+          label: 'Text',
+          value: 'text-xs',
+        },
+      ],
+      size: 'xs',
+      selectedButtonValue: 'label-xs',
+      onChange: value => console.log('clicked: ', value),
+    },
+    {
+      buttons: [
+        {label: 'Label', value: 'label-s'},
+        {
+          label: 'Another label',
+          value: 'another-label-s',
+        },
+        {
+          label: 'Text',
+          value: 'text-s',
+        },
+      ],
+      size: 's',
+      selectedButtonValue: 'another-label-s',
+      onChange: value => console.log('clicked: ', value),
+    },
+    {
+      buttons: [
+        {label: 'Label', value: 'label-m'},
+        {
+          label: 'Another label',
+          value: 'another-label-m',
+        },
+        {
+          label: 'Text',
+          value: 'text-m',
+        },
+      ],
+      size: 'm',
+      selectedButtonValue: 'text-m',
+      onChange: value => console.log('clicked: ', value),
+    },
+    {
+      buttons: [
+        {label: 'Label', value: 'label-l'},
+        {
+          label: 'Another label',
+          value: 'another-label-l',
+        },
+        {
+          label: 'Text',
+          value: 'text-l',
+        },
+      ],
+      size: 'l',
+      selectedButtonValue: 'label-l',
+      onChange: value => console.log('clicked: ', value),
+    },
+  ],
+};
+
+export const TypesOfSegmentedButtons = MultipleTemplate.bind({});
+TypesOfSegmentedButtons.args = {
+  components: [
+    {
+      buttons: [
+        {label: 'Label', value: 'label'},
+        {
+          label: 'Label Two',
+          value: 'label-two',
+        },
+        {
+          label: 'Label Three',
+          value: 'label-three',
+        },
+      ],
+      type: 'withLabel',
+      selectedButtonValue: 'label',
+      onChange: value => console.log('clicked: ', value),
+    },
+    {
+      buttons: [
+        {
+          label: 'Label',
+          value: 'label-il',
+          iconLeft: {iconName: 'check', iconStyle: 'solid', title: 'check'},
+        },
+        {
+          label: 'Label Two',
+          value: 'label-two-il',
+          iconLeft: {iconName: 'check', iconStyle: 'solid', title: 'check'},
+        },
+        {
+          label: 'Label Three',
+          value: 'label-three-il',
+          iconLeft: {iconName: 'check', iconStyle: 'solid', title: 'check'},
+        },
+      ],
+      type: 'withLabel',
+      selectedButtonValue: 'label-il',
+      onChange: value => console.log('clicked: ', value),
+    },
+    {
+      buttons: [
+        {
+          label: 'Label',
+          value: 'label-ir',
+          iconRight: {iconName: 'check', iconStyle: 'solid', title: 'check'},
```

Comment:
You could use a different icon so folks know if just looking at the storybook that the icon is not always a checkmark when used with a text label.

Reply:
Good point, thanks. Will fix